### ISSUE TITLE
ERROR: Check failed: error == cudaSuccess (48 vs. 0) no kernel image is available for execution on the device

### ISSUE THREAD

**Issue by DennisFaucher** [dusty-nv/jetson-inference#627]
Hi. Spun up an AWS GPU image (Deep Learning AMI (Ubuntu 18.04) Version 29.0 - ami-043f9aeaf108ebc37
MXNet-1.6.0, Tensorflow-2.1.0 & 1.15.2, PyTorch-1.4.0 & 1.5.0, Neuron, & other frameworks, NVIDIA CUDA, cuDNN, NCCL, Intel MKL-DNN, Docker, NVIDIA-Docker & EFA support. For fully managed experience, check: https://aws.amazon.com/sagemaker on g2.2xlarge)

Installed the DIGITS container image and successfully started it and created my test/val data set. 

docker login nvcr.io
Username: $oauthtoken
Password: [long_string]

docker run --runtime=nvidia --rm nvcr.io/nvidia/cuda:9.0-cudnn7-devel-ubuntu16.04 nvidia-smi

When I went to create my Object Recognition model, I ran into to issues:

1) There are no pre-trained models listed (Instructions say to select GoogleNet)
2) Running "Create Model" failed with these erroros

ERROR: Check failed: error == cudaSuccess (48 vs. 0) no kernel image is available for execution on the device

This network produces output recall
Top memory (TEST) required for data: 1285500056 diff: 1285500056
Bottom memory (TEST) required for data: 1285500016 diff: 1285500016
Shared (in-place) memory (TEST) by data: 272203776 diff: 272203776
Parameters memory (TEST) required for data: 23914712 diff: 23914712
Parameters shared memory (TEST) by data: 0 diff: 0
Network initialization done.
Solver scaffolding done.
Starting Optimization
Solving
Learning Rate Policy: step
Reserving 23918336 bytes of shared learnable space for type FLOAT
Initial Test started...
Iteration 0, Testing net (#0)
Ignoring source layer train_data
Ignoring source layer train_label
Ignoring source layer train_transform
NVML initialized, thread 376
Check failed: error == cudaSuccess (48 vs. 0)  no kernel image is available for execution on the device
NVML succeeded to set CPU affinity on device 0, thread 376
        
What did I do wrong? TIA

**Comment by DennisFaucher** [comment]
I'm going to close this. I decided to use the AWS NVIDIA DIGITS Marketplace image instead https://aws.amazon.com/marketplace/pp/B01LZN28VD
